From 64f7788cfec7da6ec896c5fcaa560acf96527467 Mon Sep 17 00:00:00 2001
From: Trevor Gamblin <tgamblin@baylibre.com>
Date: Wed, 10 Sep 2025 13:23:31 -0400
Subject: [PATCH] setup.py: update to match manylinux 2_39

Upstream-Status: Inappropriate [upstream needs larger fix for manylinux support]

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 89cde3d..470422a 100644
--- a/setup.py
+++ b/setup.py
@@ -45,7 +45,7 @@ else:
         """Create OS-dependent, but Python-independent wheels."""
 
         def get_tag(self):
-            pythons = 'py2.py3'
+            pythons = 'py3'
             if platform == 'darwin':
                 if architecture0 == 'x86_64':
                     oses = 'macosx_10_9_x86_64'
@@ -65,9 +65,9 @@ else:
                 else:
                     pep600_architecture = architecture0
 
-                oses = 'manylinux_2_28_{}'.format(pep600_architecture)
+                oses = 'manylinux_2_39_{}'.format(pep600_architecture)
             else:
-                pythons = 'py2.py3'
+                pythons = 'py3'
                 oses = 'any'
             return pythons, 'none', oses
 
